README.md

```diff
@@ -1,26 +1,48 @@
-# "SkotOS" Skotos Open Source Game Library
+# SkotOS Open Source Game Library
 
-This is a pre-release version of the Skotos Open Source game library, aka "SkotOS", written for use with [Dworkin's Game Driver](https://github.com/dworkin/dgd), aka "DGD". It a fourth-generation text MUD engine that has been in devopment and active use for almost two decades by [Skotos Tech](http://www.skotos.net/) in a number of commercial web-centric prose online games, including [Castle Marrach](http://www.skotos.net/games/marrach/), [Ironclaw Online](http://www.skotos.net/games/ironclaw/), [Lovecraft Country](http://www.lovecraftcountry.com/comic/), and [The Lazurus Project](http://www.lazarus-project.net/). 
+[MUDs](https://en.wikipedia.org/wiki/MUD) were some of the earliest multiplayer games on the Internet, and in some cases before the Internet. Players would read text descriptions and type text commands to control their characters.
+
+[Skotos Tech](https://www.skotos.net/), from 1999 to 2020, was a company that made MUDs and MUD-like games "that were more social, more dynamic, more interactive, more realistic, and more story-focused than anything that had been seen before." They were a critical success but never a commercial powerhouse. MUDs never made a huge amount of money, even the successful ones.
+
+But the massively-multiplayer text interactive fiction world's loss is your gain &mdash; when SkotOS's business failed, they left behind their twenty-year game engine as open source, under the name [ChatTheatre](https://chattheatre.com).
```

Comment:
I'm not sure that we failed (we were slightly above break-even, but not enough give a return on investment to the original investors), but we didn't have a true success either, and rather than face inevitable atrophy we decided to open form a new legal entity for our underlying tech to try to leverage what it using open source approaches.

We don't need to say that here, but "business failed" doesn't seem quite right.

Reply:
That's fair. I've tried another phrasing. Would also love Shannon's feedback.